pkg/driver/driver.go

```
@@ -243,9 +244,14 @@ func (driver *Driver) AddControllerServiceCapabilities(capabilities []csi.Contro
 func (driver *Driver) AddNodeServiceCapabilities(capabilities []csi.NodeServiceCapability_RPC_Type) {
 	var nodeServiceCapabilities []*csi.NodeServiceCapability
 
+	disableNodeGetVolumeStats, _ := strconv.ParseBool(os.Getenv(disableNodeGetVolumeStatsKey))
```

nit: please return the error in case user enters non boolean flag.